core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -17,6 +17,7 @@
  * in Protractor tests.
  */
 
+const dragAndDrop = require('html-dnd').code;
```

Comment:
maybe we should let it be a ```var``` to follow the convention below. And in the future, we can replace all the ```var'``` at once. @nithusha21 @U8NWXD

Reply:
Done in 4878691de